advanced03-AF_XDP/af_xdp_kern.c

```diff
@@ -18,6 +19,17 @@ struct bpf_map_def SEC("maps") xdp_stats_map = {
 	.max_entries = 64,
 };
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,3,0)
```

Comment:
It is not the right approach to check against a kernel version.
(Because distros will backport changes)

Reply:
I know, but due to the lack of any run-time checks, what would you suggest?

Reply:
Instead we have to write a bpf probe that can detect this (like bpftool does).
@tohojo have added a [configure](https://github.com/xdp-project/xdp-tools/blob/master/configure) script in his XDP-tools repo.
In this case, we likely need to confine the check to the ```advanced03-AF_XDP/``` directory, and we can likely add it as part of the Makefile.

Reply:
The configure script just sets some variables; so we can do that in the top-level and only react to them in advanced03. I've been meaning to port over some of the build stuff I did in xdp-tools to this repo anyway, so this sounds like a way forward :)